### ISSUE TITLE
Some small mistakes

### ISSUE THREAD

**Issue by Levigty** [PatrickHua/SimSiam#42]
Thanks for your work. I thank the "_nn.BatchNorm1d(hidden_dim)_" in line 42 in the **models/simsiam.py** should be changed to "_nn.BatchNorm1d(out_dim)_".
